### PR TITLE
Increment minimum version - this prevents IDE warnings and brings us into 2016

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
@totten

### CODE REVIEW

**Comment by MikeyMJCO** [comment]
We should go further than this. Really we could/should peg this straight to `5.6`. [I already did for `cv`...](https://github.com/civicrm/cv/blob/master/composer.json)

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "bin-dir": "bin"
   },
   "require": {
-    "php": ">=5.3.3",
+    "php": ">=5.4",
```
Comment (MikeyMJCO) [inline composer.json:18]:
```suggestion
    "php": ">=5.6.0",
```

**Review by MikeyMJCO — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (eileenmcnaughton) [reply composer.json:18]: @MikeyMJCO I'd be happy with 7.1 - but at least this gets past the immediate issue...

@totten can you merge or merge the number you are happy to go to?

Comment (MikeyMJCO) [reply composer.json:18]: So would I ;-)

5.6.0 aligns it with CV's minimum version (which is included in this composer.json)

Comment (totten) [reply composer.json]: Yeah, I think 5.6 is good for now. (It's rare, but I did recently have to reproduce something where the configuration could only be initialized in PHP 5 + CIvi 4...) IMHO, we should bump this to php7 in a few months.

Also, the `composer.lock` needs  a small twiddling (`composer update --lock`).

Pushed up both those bits.

**Pull request merged**